CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# How to contribute
+
+The following is a set of guidelines for contributing to the CRDCH model repository.
+
+## Development Environment Setup
+
+This section describes how to set up the development environment needed to make the CRDCH artifacts from the CRDCH schema in YAML that is included with this repository.
+
+You will need to install `Python 3.7+`, `Make` and `Poetry` in order to build the artifacts in this repository. One-time installation commands for Poetry are available for [macOS/Linux/Bash on Windows](https://github.com/python-poetry/poetry#osx--linux--bashonwindows-install-instructions) and for [Windows PowerShell](https://github.com/python-poetry/poetry#windows-powershell-install-instructions).
+
+For most development tasks, you should be able to clone this repository using:
+
+```shell
+git clone https://github.com/cancerDHC/ccdhmodel.git
+```
+
+And then building all artifacts by using the included Makefile:
+
+```shell
+make
+```
+
+You can run the included unit tests by running:
+
+```shell
+make test
+```
+
```

Comment:
I've just realized that we don't mention that you can regenerate the model by using:

```shell
make generate-model
```

Could you please add that here?

Reply:
Sure. It is easy to add.